code/scripting/api/libs/hud.cpp

```diff
@@ -237,6 +240,29 @@ ADE_FUNC(flashTargetBox, l_HUD, "enumeration section, [number duration_in_millis
 	return ADE_RETURN_NIL;
 }
 
+ADE_FUNC(getTargetDistance, l_HUD, "object targetee, [vector targeter_pos]", "Returns the distance as displayed on the HUD, that is, the distance from a position to the bounding box of a target.  If targeter_pos is nil, the function will use the player's position.", "number", "The distance, or nil if invalid")
```

Comment:
Thoughts on changing this to `targeter_position` in both cases? Seems like, in general, that full name descriptions are a bit easier to read and more common in the scripting variable names.

Reply:
Yeah that makes sense.

Reply:
Well, in the documentation anyway.  In the code, I kept it as-is to be consistent with other usages.

Reply:
Ah yeah just the documentation was what I was noting.

Reply:
Ok.  Got the other one in the documentation too.